.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -4,24 +4,52 @@ about: Request our thoughts on your suggestion for a new feature for Homebrew.
 
 ---
 
-# Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
+<details>
+<summary><strong>Guidelines</strong></summary>
+Thank you for making a feature suggestion! :heart:
 
-<!-- Please fill these sections with the relevant information: -->
+**Please fill out all the sections below with the relevant information.**
 
-# A detailed description of the proposed feature
+We will review your suggestion and let you know our decision as soon as we can.
+Note, that we will close this issue if neither you nor we have the time to work on it.
```

Comment:
We will close it unless maintainers actively plan on working on it or making it `help wanted`.